src/Containers/PanelListWrapper/PanelListWrapper.tsx

```diff
@@ -0,0 +1,80 @@
+import { MainInterface, Main } from '@Utils/BaseStyles';
+import React, { useEffect, useState, useMemo, useRef } from 'react';
+import styled from 'styled-components';
+import { PanelCard, IPanelCardProps } from '../PanelCard/PanelCard';
+
+const DELAY_IN_MOUNTING_IN_MS = 100;
+
+export interface IPanelListWrapperProps
+    extends MainInterface,
+        React.HTMLAttributes<HTMLDivElement> {
+    panels: IPanelCardProps[];
+    /** vertical space between panels in px */
+    verticalSpacing?: number;
+    /** if true panels are added sequentially to the list */
+    isAnimatedPanels?: boolean;
+}
+
+export const PanelListWrapper: React.FC<IPanelListWrapperProps> = ({
+    panels,
+    verticalSpacing,
+    isAnimatedPanels = false,
+    ...props
+}) => {
+    const internalPanels = useAnimatedPanels(panels);
+    const panelsToMap = useMemo(
+        (): IPanelCardProps[] => (isAnimatedPanels ? internalPanels : panels),
+        [isAnimatedPanels, internalPanels, panels],
+    );
+
+    return (
+        <PanelListWrapperBox {...props}>
+            {panelsToMap.map((panel) => (
+                <PanelCard
+                    key={`${panel.name}`}
+                    {...panel}
+                    margin={`${verticalSpacing}px 0`}
+                />
+            ))}
+        </PanelListWrapperBox>
+    );
+};
+
+const PanelListWrapperBox = styled.div<MainInterface>`
+    ${(props) => Main({ ...props })}
+`;
+
+const useAnimatedPanels = (panels:IPanelCardProps[]) => {
```

Comment:
JSDOC here explaining in English the Logic Behaviour you are applying
Load Panels one by one with a Fade In Animation.
Load Panels one by one with a Fade Out Animation